.github/workflows/main.yml

```diff
@@ -38,8 +38,8 @@ jobs:
     - name: Set Python user directory
       run: echo "::set-env name=USER_DIR::$(python -c 'import os,site;print(os.path.dirname(site.USER_SITE))', end='')"
 
-    - name: Install Windows 10 SDK
-      run: choco install windows-sdk-10.0
+    #- name: Install Windows 10 SDK
```

Comment:
Thanks! I've already cherry-picked this comment (although I removed the lines entirely) because we might as well get this in asap!